.github/workflows/pr-auto.yml

```
@@ -17,16 +17,25 @@ jobs:
-          # PR 제목에 따라 라벨 자동 지정
-          labels: |
-            ${{ github.event.pull_request.title.startsWith('[fix]') && 'bug' || 
-               github.event.pull_request.title.startsWith('[chore]') && 'chore' || 
-               github.event.pull_request.title.startsWith('[feat]') && 'feature' || 
-               github.event.pull_request.title.startsWith('[refactor]') && 'refactor' }}
+        with: { labels: bug }
+
+      - name: 'chore' 라벨 추가
+        if: startsWith(github.event.pull_request.title, 'Chore') || startsWith(github.event.pull_request.title, 'chore')
+        uses: actions-ecosystem/action-add-labels@v1
+        with: { labels: chore }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)